Tools/HelperScripts/static_analysis_files.bash

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# define path to the tools dir
+TOOLS_DIR=$(dirname -- "$0")/..
+JSON_PATH=$TOOLS_DIR/Firmware/build/compile_commands.json
```

Comment:
This file only exists if the if the '# create build system' section of 'build.bash' is run first. What happens if this file does not exist?

Reply:
Added a check for this. If that file doesn't exist the the linting script stops and tells the user to build the project first.